ci/deploy.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash -e
+#
+# Deploy your branch on VIP Go.
+#
+
+set -ex
+
+DEPLOY_SUFFIX="-built"
+
+BRANCH="${CIRCLE_BRANCH:-$TRAVIS_BRANCH}"
+SRC_DIR="${TRAVIS_BUILD_DIR:-$PWD}"
```

Comment:
I'm assuming we'd have to manually change these on Circle CI?

Reply:
I'm not sure what you mean. I've added a comment explaining the `CIRCLE_*` and `TRAVIS_*` environment variables are set by the CI provider. 

Looking into `$PWD`, it seems to be a somewhat reliable variable set by the shell. I could move to `PWD=$(pwd)` for more safety.

…or are you thinking of something else?

Reply:
Oh, I missed `CIRCLE_BRANCH` :|